python/pyspark/sql/functions/builtin.py

```diff
@@ -12912,6 +12912,108 @@ def substr(
         return _invoke_function_over_columns("substr", str, pos)
 
 
+@_try_remote_functions
```

Comment:
can we add a one unittest in, e.g., test_functions.py? then the tests will be resued in both Spark connect and classic.